### ISSUE TITLE
how do i use this with a web server?

### ISSUE THREAD

**Issue by laurencefass** [INsReady/docker-drupal-dev#3]
support question

is there a web server in the container?
if not please can you offer best suggestion to run it with a web server?
please can you suggest a run command for volume shares etc?

thanks

**Comment by skyred** [comment]
> is there a web server in the container?

yes. It's Apache, because this image inherits the official image https://hub.docker.com/_/drupal


> please can you suggest a run command for volume shares etc?

See the "Volumes" section on the official doc: https://hub.docker.com/_/drupal
